public/javascripts/index.js

```diff
@@ -43,15 +30,27 @@ const Letter = React.createClass({
     }
 })
 
-const Alphabet = React.createClass({
+const ButtonChoices = React.createClass({
+    showCapitalA() {
+	console.log("in show")
+	return <Letter />
+    },
+    
     render() {
 	return (
-	    <div id="a">
-	    <ButtonChoices />
-	    <Letter />
+	    <div id="buttons">
+	    <div id="button-group">
+	    <div id="capital-button" onClick={this.showCapitalA}>A</div>
```

Comment:
So, when a user clicks this button, we'll run `this.showCapitalA`, which is a function that just happens to return a component. It doesn't _do_ anything with that component does. It just returns it. We're not doing anything with that returned value, so it just gets garbage collected.

Now, if we wanted it to do something (which I suspect we do) we could instead modify state and announce that "A" is now the currently selected letter.